src/app/lib/config/services/ukrainian.js

```diff
@@ -4,277 +4,289 @@ import { ukrainian as brandSVG } from '@bbc/psammead-assets/svgs';
+        text: 'Пропустити %provider_name% допис',
+        endTextVisuallyHidden: 'Кінець %provider_name% допису',
       },
+    },
+    topStoriesTitle: 'Top Stories',
+    featuresAnalysisTitle: 'Докладно',
+  },
+  brandSVG,
+  mostRead: {
+    header: 'Найпопулярніше',
+    lastUpdated: 'Останнє оновлення:',
+    numberOfItems: 10,
+    hasMostRead: true,
+    onIdxPage: false,
+  },
+  radioSchedule: {
+    hasRadioSchedule: false,
+    onIdxPage: false,
+  },
+  recommendations: {
+    hasStoryRecommendations: false,
+  },
+  footer: {
+    trustProjectLink: {
+      href: 'https://www.bbc.com/ukrainian/institutional-50170368',
+      text: 'Чому BBC заслуговує на довіру',
+    },
+    externalLink: {
+      href: 'https://www.bbc.co.uk/help/web/links/',
+      text: 'Ознайомтеся з нашими правилами зовнішніх посилань.',
+    },
+    links: [
       {
-        title: 'Здоров’я',
-        url: '/ukrainian/topics/c4794229-7f87-43ce-ac0a-6cfcd6d3cef2',
+        href: 'https://www.bbc.com/ukrainian/institutional-38144387',
+        text: 'Правила користування',
       },
       {
-        title: 'Журнал',
-        url: '/ukrainian/magazine',
+        href: 'https://www.bbc.com/ukrainian/institutional-38144827',
+        text: 'Особисті дані',
       },
       {
-        title: 'Книга року ВВС',
-        url: '/ukrainian/features-50320117',
+        href: 'https://www.bbc.com/usingthebbc/cookies/',
+        text: 'Cookies',
       },
       {
-        title: 'Блоги',
-        url: '/ukrainian/blogs',
+        href: 'https://www.bbc.com/ukrainian/institutional-38146278',
+        text: "Зв'язатися з BBC",
       },
       {
-        title: 'РУС',
-        url: '/ukrainian/ukraine_in_russian',
+        href:
+          'https://www.bbc.com/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
+        text: 'AdChoices / Do Not Sell My Info',
+        lang: 'en-GB',
       },
     ],
+    copyrightText: 'BBC. ВВС не несе відповідальності за контент інших сайтів.',
+  },
+  fonts: [],
+  timezone: 'GMT',
+  navigation: [
+    {
+      title: 'Новини',
+      url: '/ukrainian',
+    },
+    {
+      title: 'Коронавірус',
+      url: '/ukrainian/52118881',
+    },
+    {
+      title: 'Наука',
+      url: '/ukrainian/topics/0f469e6a-d4a6-46f2-b727-2bd039cb6b53',
+    },
+    {
+      title: 'Технології',
+      url: '/ukrainian/topics/31684f19-84d6-41f6-b033-7ae08098572a',
+    },
+    {
+      title: 'Здоров’я',
+      url: '/ukrainian/topics/c4794229-7f87-43ce-ac0a-6cfcd6d3cef2',
+    },
+    {
+      title: 'Журнал',
+      url: '/ukrainian/magazine',
+    },
+    {
+      title: 'Книга року ВВС',
+      url: '/ukrainian/features-50320117',
+    },
+    {
+      title: 'Блоги',
+      url: '/ukrainian/blogs',
+    },
+    {
+      title: 'РУС',
+      url: '/ukrainian/ukraine_in_russian',
+    },
+  ],
+};
+
+export const service = {
+  default: {
+    ...baseServiceConfig,
+    datetimeLocale: 'uk',
+    locale: 'uk-UA',
```

Comment:
I believe locales should have underscores. 
```suggestion
    locale: 'uk_UA',
```